CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Changelog
 
+### Web3
+
+- Added new design for NFT spam management in Brave Wallet. ([#8205](https://github.com/brave/brave-ios/issues/8205))
+- Added bottom tab bar for Brave Wallet. ([#8166](https://github.com/brave/brave-ios/issues/8166))
+- Added the ability to group NFTs in Brave Wallet. ([#8073](https://github.com/brave/brave-ios/issues/8073))
+- Added new design for the Portfolio page. ([#8072](https://github.com/brave/brave-ios/issues/8072))
+- Added method to return "Swap" protocol fees. ([#7924](https://github.com/brave/brave-ios/issues/7924))
+- Added support for Sign-In with Ethereum (EIP-4361). ([#7827](https://github.com/brave/brave-ios/issues/7827))
+- Added support for EIP-6963. ([#7779](https://github.com/brave/brave-ios/issues/7779))
+- Added the ability for users to navigate to ENS domains. ([#6225](https://github.com/brave/brave-ios/issues/6225))
+- Added the ability to select a specific RPC URL as active for Brave Wallet. ([#5902](https://github.com/brave/brave-ios/issues/5902))
+- [Security] Updated Brave Wallet password policy. ([#6093](https://github.com/brave/brave-ios/issues/6093))
+- Implemented "Bridge to Aurora" for Brave Wallet. ([#5785](https://github.com/brave/brave-ios/issues/5785))
+- Updated Portfolio settings page in Brave Wallet. ([#8089](https://github.com/brave/brave-ios/issues/8089))
+- Updated button to create wallet to be disabled when wallet is already being created. ([#8357](https://github.com/brave/brave-ios/issues/8357))
+- Updated “Send” screen in Brave Wallet to not show decimal values for zero balances. ([#8232](https://github.com/brave/brave-ios/issues/8232))
+- Updated “Save Changes” button to be enabled only when there is a change in wallet portfolio settings. ([#8229](https://github.com/brave/brave-ios/issues/8229))
+- Fixed “view my recovery phrase” not displaying in Brave Wallet. ([#8445](https://github.com/brave/brave-ios/issues/8445))
+
+### General 
+
+- Added support for all datatypes when "History" sync is enabled. ([#8305](https://github.com/brave/brave-ios/issues/8305))
```

Comment:
This should be indicating all history data is synced when history sync is enabled not just type urls. With this title, it feels like other sync types are enabled.